samples/snippets/list_secret_versions_with_filter.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+"""
+command line application and sample code for listing secret versions of a
+secret.
+"""
+
+import argparse
+
+
+# [START secretmanager_list_secret_versions_with_filter]
+def list_secret_versions_with_filter(project_id, secret_id, filter_str):
```

Comment:
Consider pre-filling with a valid `filter_str = "..."` to help users. The `state:ENABLED` passed in the test seems like a good option.

Reply:
Done.